raiden/tests/utils/transfer.py

```diff
@@ -897,7 +897,7 @@ def wait_assert(func: Callable, *args, **kwargs) -> None:
             func(*args, **kwargs)
         except AssertionError as e:
             try:
-                gevent.sleep(0.5)
+                gevent.sleep(0.001)
```

Comment:
If func does something slow (e.g. a request to a status endpoint), executing it 1000/s could be a bit much. I didn't find any such cases though, so I assume it's ok.

Reply:
Yeah, I thought about that, but in the end decided to just go with this as the alternative would be to to have a new variant of `wait_assert`, which I was not terribly happy about.